### PR TITLE
[1866] Add all routes to the "what type of training" form

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Adding all of the early years routes to the `What type of training` form. 

When you select one of the new routes, it now redirects to the correct form.

`Note:` Two routes have been renamed in this PR as detailed below. 

### Changes proposed in this pull request

- All routes now redirect to the correct form
- Early years routes are switched on in the review app

- Rename "Early years (graduate employment based)" to "Early years (salaried)"
- Rename "Early years (graduate entry)" to "Early years (postgrad)"

### Guidance to review

- Add a new trainee 
- Check that choosing an early year route redirects to the correct form
- Repeat with the other early year routes

### CODE REVIEW

**Comment by avinhurry** [comment]
[[S] EY - add all routes to 'What type of training?' form](https://trello.com/c/jA7qrmpn/1866-s-ey-add-all-routes-to-what-type-of-training-form)

**Review by AbigailMcP — COMMENTED** [review]
Nice! All working correctly in review apps.

There are some discrepancies with names vs the prototype though:

Screenshot 2021-06-02 at 17 17 57

Please can you check with @edwardhorsford and update? Thanks!

**Comment by edwardhorsford** [comment]
> Nice! All working correctly in review apps.
> 
> There are some discrepancies with names vs the prototype though:
> 
> Screenshot 2021-06-02 at 17 17 57
> 
> Please can you check with @edwardhorsford and update? Thanks!

Good catch! I actually only changed them [this morning](https://github.com/DFE-Digital/register-trainee-teachers-prototype/pull/277) - if we can update now we should, otherwise we can make a separate card for it.

File: config/settings/development.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ features:
     provider_led_postgrad: true
     early_years_undergrad: true
     school_direct_salaried: true
-    school_direct_tuition_fee: true
+    school_direct_fee_funded: true
```
Comment (AbigailMcP) [inline config/settings/development.yml:13]: to be removed!

Comment (avinhurry) [reply config/settings/development.yml]: Oops. Done.

**Review by AbigailMcP — APPROVED** [review]

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**